.github/workflows/on-pull-request.yml

```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [16.x, 18.x]
```

Comment:
IMHO we should support all node 14 to.

Reply:
It's reverted, we'll see if the workflows pass

Reply:
Unfortunately the workflow fails with node 12 and 14 for an unknown reason, ok to remove it?

Reply:
I'd prefer to have a node 14 run to. But as it's just about the build, it's not that important. So feel free to merge.